packages/semcom-components/lib/components/sem-com-register.component.ts

```diff
@@ -0,0 +1,123 @@
+import { html, css, CSSResult, TemplateResult, state, unsafeCSS, query } from 'lit-element';
+import { RxLitElement } from 'rx-lit';
+import { from, Observable, ObservableInput } from 'rxjs';
+import { createMachine, interpret, State } from 'xstate';
+import { Theme } from '@digita-ai/ui-transfer-theme';
+import { SolidSDKService } from '@digita-ai/ui-transfer-components';
+import { semComRegisterMachine, SemComRegisterContext, SemComRegisterEvent, SemComRegisterState, SemComRegisterStates, AuthenticatedEvent, BackToStoreSelectionEvent, BackToUploadFormEvent } from './sem-com-register.machine';
+
+export class SemComRegisterComponent extends RxLitElement {
+
+  private solidService = new SolidSDKService('SemCom');
+
+  private machine = createMachine<SemComRegisterContext, SemComRegisterEvent, SemComRegisterState>(
+    semComRegisterMachine
+  );
+
+  // Should not be an any - due to ts strict mode.
+  private actor: any = interpret(this.machine, { devTools: true })
```

Comment:
No need for a type here, I think, let TS just infer it.

Reply:
All of the comments that mention `- due to ts strict mode` need to be together for everything to work, it seems to have a cascading effect. In ui-transfer repo I do not have a type here, but strictmode causes an error on line 33 when subscribing to the state with `from(this.actor)`. If I dont have the any on the actor, and I dont cast on line 33, it doesn't work.

Error without the any but WITH the cast on line 33 (the error occurse on line 33 in the `from(this.actor)`):
```
Argument of type 'Interpreter<SemComRegisterContext, any, SemComRegisterEvent, SemComRegisterState>' is not assignable to parameter of type 'ObservableInput<any>'.
  Type 'Interpreter<SemComRegisterContext, any, SemComRegisterEvent, SemComRegisterState>' is not assignable to type 'InteropObservable<any>'.
    The types of '[Symbol.observable]().subscribe' are incompatible between these types.
      Type '{ (next: (value: State<SemComRegisterContext, SemComRegisterEvent, any, SemComRegisterState>) => void, error?: ((error: any) => void) | undefined, complete?: (() => void) | undefined): Subscription; (observer: Observer<...>): Subscription; }' is not assignable to type '{ (observer?: PartialObserver<any> | undefined): Unsubscribable; (next: null | undefined, error: null | undefined, complete: () => void): Unsubscribable; (next: null | undefined, error: (error: any) => void, complete?: (() => void) | undefined): Unsubscribable; (next: (value: any) => void, error: null | undefined, c...'.
        Types of parameters 'next' and 'next' are incompatible.
          Type 'null | undefined' is not assignable to type '(value: State<SemComRegisterContext, SemComRegisterEvent, any, SemComRegisterState>) => void'.
            Type 'undefined' is not assignable to type '(value: State<SemComRegisterContext, SemComRegisterEvent, any, SemComRegisterState>) => void'.
```